app/src/main/java/com/f0x1d/logfox/extensions/views/ViewExtensions.kt

```diff
@@ -1,6 +1,16 @@
 package com.f0x1d.logfox.extensions.views
 
 import android.view.View
+import android.view.accessibility.AccessibilityNodeInfo
 import com.google.android.material.snackbar.Snackbar
 
-fun View.snackbar(text: String) = Snackbar.make(this, text, Snackbar.LENGTH_SHORT).show()
\ No newline at end of file
+fun View.snackbar(text: String) = Snackbar.make(this, text, Snackbar.LENGTH_SHORT).show()
+
+fun View.setAccessibilityDelegateForClassName(clazz: Class<out View>) {
```

Comment:
I don't really like the naming. It's like we don't set **FOR** class name. We mostly replace our current behavior with another behavior if I understand correctly

Reply:
Yes, but I can't think of a better name for it. the MaterialComponents approach is to rewrite the View. but I'm not sure at this point if you'd like to rewrite the View to support this feature. Because we could probably try to adapt MaterialButton to achieve a similar effect

Reply:
No, of course this is not the thing to pay so much attention to. What about `replaceAccessibilityDelegateClassNameWith(clazz: Class<out View>)`?

Reply:
I think this is also a bit inappropriate because this method will overwrite the original object.

Reply:
Then what's wrong? Replace somehow means overwrite